### PR TITLE
HPCC-17457 Add instructions for debugging java services

### PR DESCRIPTION
**Pull request by mayx**
- Add a commented-out line to environment.conf for java remote debugging
- Add debugging instructions in ReadMeFirst.txt file of esdl examples

Signed-off-by: mayx <yanrui.ma@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [x] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [x] Performance
  - [x] Security
  - [x] Thread-safety
  - [x] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by mayx** [comment]
@afishbeck Hi Tony please review the change.

File: initfiles/examples/EsdlExample/ReadMeFirst.txt [hunk 1]
```diff
@@ -52,3 +52,9 @@ soapplus -url http://.:8088/EsdlExample -i roxierequest.xml
 
 http://<DynamicEsdlIP>:8088
 
+9. To debug the java service:
+Uncomment the jvmoptions line in environment.conf:
+jvmoptions=-XX:-UsePerfData -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=2000
+And comment out the original jvmoptions line:
+#jvmoptions=-XX:-UsePerfData
+From eclipse for java, create a new debug configuration for remote java application, under connect tab, select your esdl java service project, select standard socket attach connection type, fill in the host and port. The port is what you specify in the jvmoptions inside environment.conf, which is 2000 in our example above. Click the Debug button and off you go.
```
Comment (afishbeck) [inline initfiles/examples/EsdlExample/ReadMeFirst.txt]: "specified" or even better, "The port should match what you've specified".

**Review by afishbeck — CHANGES_REQUESTED** [review]
@mayx minor wording comment

**Comment by mayx** [comment]
@afishbeck Wording updated as suggested, please review again...

**Review by afishbeck — APPROVED** [review]
@mayx looks good

**Pull request merged**